xarray/core/formatting_html.py

```diff
@@ -0,0 +1,312 @@
+# coding: utf-8
+
+import uuid
+import pkg_resources
+from functools import partial
+from collections import OrderedDict
+
+from .formatting import format_array_flat
+
+
+CSS_FILE_PATH = '/'.join(('static', 'css', 'style-jupyterlab.css'))
```

Comment:
We could probably just call this `style.css`, since we're going to use it everywhere, even outside of jupyterlab.